src/Modules/AuthModule/components/Login/Login.tsx

```diff
@@ -1,144 +1,142 @@
 import { FormData } from "../../../../interfaces/Auth";
 import { useUser } from "../../../../Context/AuthContext";
 import { useToast } from "../../../../Context/ToastContext";
 export default function Login() {
-  // All states here on the top
-  const [showPassword, setShowPassword] = useState<boolean>(false);
-  const { showSuccessToast, showErrorToast } = useToast();
+	// All states here on the top
+	const [showPassword, setShowPassword] = useState<boolean>(false);
+	const { showSuccessToast, showErrorToast } = useToast();
 
-  const {saveAdminData , adminData} = useUser();
-  console.log(adminData)
-  const [spinner, setSpinner] = useState<boolean>(false);
-  const {
-    register,
-    handleSubmit,
-    formState: { errors },
-  } = useForm<FormData>();
-  const navigate = useNavigate();
-  // test done
-  // Function for password visibility
-  //  void indicates that a function does not return any value
-  const togglePasswordVisibility = (): void => {
-    setShowPassword((prevState: boolean) => !prevState);
-  };
+	const { saveAdminData, adminData } = useUser();
+	console.log(adminData);
+	const [spinner, setSpinner] = useState<boolean>(false);
+	const {
+		register,
+		handleSubmit,
+		formState: { errors },
+	} = useForm<FormData>();
+	const navigate = useNavigate();
+	// test done
+	// Function for password visibility
+	//  void indicates that a function does not return any value
+	const togglePasswordVisibility = (): void => {
+		setShowPassword((prevState: boolean) => !prevState);
+	};
 
-  // senD Data to Api
-  const onSubmit = async (data: FormData) => {
-    setSpinner(true);
+	// senD Data to Api
+	const onSubmit = async (data: FormData) => {
+		setSpinner(true);
 
-    try {
-      const response = await axios.post(
-        "https://upskilling-egypt.com:3003/api/v1/Users/Login",
-        data
-      );
-      localStorage.setItem("adminToken", response?.data?.token);
-      showSuccessToast("Login successfully");
-      localStorage.setItem("token", response.data.token);
-      navigate("/dashboard");
-      saveAdminData && saveAdminData(); // Call saveAdminData only if it exists
-    } catch (error) {
-      if (axios.isAxiosError(error) && error.response) {
-        showErrorToast(error.response.data.message)}
-        else {
-          // Handle other types of errors here
-          showErrorToast("An error occurred.");
-        }
-    } finally {
-      setSpinner(false);
-    }
-  };
+		try {
+			const response = await axios.post(
+				"https://upskilling-egypt.com:3003/api/v1/Users/Login",
+				data
+			);
+			localStorage.setItem("adminToken", response?.data?.token);
+			showSuccessToast("Login successfully");
+			localStorage.setItem("token", response.data.token);
+			navigate("/dashboard");
+			saveAdminData && saveAdminData(); // Call saveAdminData only if it exists
+		} catch (error) {
+			if (axios.isAxiosError(error) && error.response) {
+				showErrorToast(error.response.data.message);
```

Comment:
From Security point,  we can make this error: "Email or Password is incorrect"

Reply:
yes, in terms of security I totally agree but error msg in both inputs or what?

Reply:
@Mandela95 
No I meant in the toaster, because the validity of the password is only possible in the backend side.